README.md

```diff
@@ -1,11 +1,37 @@
 # Temporal in Rust
 
-Provides a standard API for working with dates and time.
+:warning: This crate is highly experimental and NOT stable. Do not use in production without extreme caution :warning:
 
-IMPORTANT NOTE: The Temporal Proposal is still in Stage 3. As such, this crate should be viewed
-as highly experimental until the proposal has been completely standardized and released.
+`Temporal` is a calendar and timezone aware date/time library that is currently being designed and proposed as a new
+builtin to the `ECMAScript` specification.
 
-## Goal
+This crate is an implementation of `Temporal` in Rust. While initially developed for the `Boa`, the crate has been externalized
+as we intended to make an engine agnostic and general usage implementation of `Temporal` and its algorithms.
 
-The intended goal of this crate is to provide an engine agnostic
-implementation of `ECMAScript`'s Temporal algorithms.
+IMPORTANT NOTE: The Temporal Proposal is still a Stage 3 proposal. As such, regardless of current state of this repository,
+please refrain from using without flags.
```

Comment:
I think all the warnings about the experimental nature of the crate can be joined into a single warning at the start of the README.